GOT-StudentVersion/GOT-StudentVersion/SeasonData.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  SeasonData.swift
+//  GOT-StudentVersion
+//
+//  Created by Sam Roman on 8/12/19.
+//  Copyright © 2019 C4Q . All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+
+class SeasonData{
```

Comment:
Good work! you created your own model to sort by the season! you can also refactor on the GOTDate like:
  private func filterSeason () -> [[GOTEpisode]] {
        arrOfSeasons = [GOTEpisode.allEpisodes.filter{$0.season == 1}, GOTEpisode.allEpisodes.filter{$0.season == 2},GOTEpisode.allEpisodes.filter{$0.season == 3},GOTEpisode.allEpisodes.filter{$0.season == 4},GOTEpisode.allEpisodes.filter{$0.season == 5},GOTEpisode.allEpisodes.filter{$0.season == 6}, GOTEpisode.allEpisodes.filter{$0.season == 7}]
        return arrOfSeasons
    }